backend/src/main/java/de/neuefische/backend/service/IdService.java

```diff
@@ -0,0 +1,9 @@
+package de.neuefische.backend.service;
+
+import java.util.UUID;
+
```

Comment:
Annotation is missing

```suggestion
@Service
```